app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'user_id'
+  has_many :likes, foreign_key: 'user_id'
+  has_many :comments, foreign_key: 'user_id'
```

Comment:
- [ ] As the previous reviewer mentioned and as per this [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), please make the foreign key that connects table users with tables posts, likes, and comments `author_id` as the last reviewer showed. and I know this is giving you an error and that's happening because you have the same problem in the migration that creates posts, likes, and comments tables https://github.com/darikmohammed/blog-app/blob/69536a1d1fc8ae2afa1328e1c2ba76d3390eed8f/db/migrate/20220914190707_create_posts.rb#L4
this line suppose to be like that
```ruby 
t.references :author, foreign_key: { to_table: 'users' }
```
so please edit all migrations i mentioned to make the foreign key `author_id`.

Reply:
@Omar-Muhamad Thanks for the point. This should have been the previous PR errors. but as of now, I am fixing it.